BuoyHealth/symptomcss
# Stylelint

This PR adds css linter to the repo. CSS files are automatically linted when they are staged. Only staged files get linted. You can also lint all files in `src` with `npm run lint`. 

## To Test

1. Checkout branch and run `npm install`
1. Ensure there aren't any style errors with `npm run lint`

If you want to test that css files are correctly linted,  force a css error in a file, ex. remove the space before curly bracket. `git add` the file you changed. Then run `git commit`. The linter should prevent you from committing the file.
